upstream-community-operators/openshift-pipelines-operator/openshift-pipelines-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Developer Tools"
+    description: OpenShift Pipelines is a cloud-native CI/CD solution for building pipelines using Tekton concepts which run natively on OpenShift and Kubernetes.
+    containerImage: quay.io/openshift-pipeline/openshift-pipelines-operator:v0.3.1-1
```

Comment:
If it's upstream(assuming the tekton) then shouldn't it refer tekton community rather RedHat? This image is the same as 
[downstream](https://github.com/operator-framework/community-operators/pull/296/files#diff-4cb1ff32039ec21b3cea67beb7c4a989R8) one, so wondering :thinking: why we need to repeat the same yaml manifests twice?

Reply:
Both of these CSV s represents the downstream operator

we added it twice (in two different directories) for two different platforms
refer: https://github.com/operator-framework/community-operators/blob/master/README.md#where-to-contribute